test/mocha/050-client-api.js

```diff
@@ -32,6 +53,24 @@ describe('Client API', () => {
       err.cause.details.should.have.property('port');
       err.cause.details.code.should.equal('ECONNREFUSED');
     });
+
+    it('should notify peer', async () => {
+      const localPeerId = peerId.localPeer.peerId;
+      const remotePeer = {
+        id: localPeerId,
+        url: localPeerId
+      };
+      let err;
+      try {
+        // notifyPeer does not return anything
+        await _client.notifyPeer({ledgerNodeId, remotePeer});
+      } catch(e) {
+        err = e;
+      }
+      should.not.exist(err);
+      // FIXME look for the actual notification here
```

Comment:
Can we address this FIXME or remove it and add an issue to improve the assertions on notification tests?